docs/codeql/writing-codeql-queries/codeql-queries.rst

```diff
@@ -25,3 +26,4 @@ CodeQL queries are used in code scanning analyses to find problems in source cod
 - :doc:`About data flow analysis <about-data-flow-analysis>`: Data flow analysis is used to compute the possible values that a variable can hold at various points in a program, determining how those values propagate through the program and where they are used. 
 - :doc:`Creating path queries <creating-path-queries>`: You can create path queries to visualize the flow of information through a codebase.
 - :doc:`Troubleshooting query performance <troubleshooting-query-performance>`: Improve the performance of your CodeQL queries by following a few simple guidelines.
+- :doc:`Debugging data-flow queries using partial flow <debugging-data-flow-queries-using-partial-flow>`: If a data-flow query unexpectedly produces no results, you can use partial flow to debug the problem.
```

Comment:
Perhaps `If a data-flow does not produce expected results` instead of `If a data-flow query unexpectedly produces no results`, as it may also be relevant to debug if a query has results. (Same change in the other file).